package/MDAnalysis/analysis/__init__.py

```diff
@@ -56,7 +56,7 @@
     Analyze hydrogen bonds, including both the per frame results as well
     as the dynamic properties and lifetimes.
 
-:mod:`~MDAnalysis.analysis.helanal`
+:mod:`~MDAnalysis.analysis.helix_analysis`
```

Comment:
Probably will want to remove the entry from `all` below?
Also the sphinx stub for helanal, and the entry in analysis_modules.rst.

Reply:
Thanks @IAlibay!